autogen/variables.tf.tmpl

```diff
@@ -85,6 +85,10 @@ variable "backends" {
       port                = number
       host                = string
     })
+    log_config = object({
```

Comment:
I'm not sure. But it seems because of https://github.com/hashicorp/terraform/issues/19898, I can't make variables optional in the block here.

If I don't include those attribute here. It seems Terraform will not recognize those attributes.

Reply:
Correct, you have to specify each value (at least setting to `null`).